### PR TITLE
Work item final train script

### PR DESCRIPTION
**Pull request by syKevinPeng**

### CODE REVIEW

File: src/final_train_script.py [hunk 1]
```diff
@@ -86,6 +86,30 @@ def training(model, train_dataloader, val_dataloader, num_epochs, lr,
...
+                pred_mask = model(img.to(DEVICE))
+                if len(result) == 0:
+                    # result = torch.unsqueeze(pred_mask, 0)
+                    result = torch.squeeze(pred_mask,1)
+                else:
+                    result = torch.cat((result, pred_mask), 0)
+    print(f'prediction result: {result.shape}')
+    sigmoid = torch.sigmoid(result)
+    sigmoid[result > 0.5] = 1
+    sigmoid[result <= 0.5] = 0
```
Comment (syKevinPeng) [inline src/final_train_script.py:110]: The output of the network is not binary class. So no sigmoid here. The output is depth estimation. Try to find some example and take a look at theirs
